plugins/woocommerce-admin/client/core-profiler/pages/BusinessLocation.tsx

```diff
@@ -1,26 +1,97 @@
 /**
- * Internal dependencies
+ * External dependencies
  */
-import { BusinessLocationEvent } from '../index';
+import { __ } from '@wordpress/i18n';
+import { Button } from '@wordpress/components';
+import { SelectControl } from '@woocommerce/components';
+import { Icon, chevronDown } from '@wordpress/icons';
+import { useState } from '@wordpress/element';
 
+/**
+ * Internal dependencies
+ */
+import {
+	BusinessLocationEvent,
+	CoreProfilerStateMachineContext,
+} from '../index';
+import { CountryStateOption } from '../services/country';
+import { Heading } from '../components/heading/heading';
+import { Navigation } from '../components/navigation/navigation';
 export const BusinessLocation = ( {
 	sendEvent,
+	navigationProgress,
+	context,
 }: {
 	sendEvent: ( event: BusinessLocationEvent ) => void;
+	navigationProgress: number;
+	context: CoreProfilerStateMachineContext;
 } ) => {
+	const [ storeCountry, setStoreCountry ] = useState< CountryStateOption >( {
+		key: '',
+		label: '',
+	} );
+
 	return (
-		<>
-			<div>Business Location</div>
-			<button
-				onClick={ () =>
-					sendEvent( {
-						type: 'BUSINESS_LOCATION_COMPLETED',
-						payload: { businessInfo: { location: 'Zanarkand' } },
-					} )
-				}
-			>
-				Next
-			</button>
-		</>
+		<div className="woocommerce-profiler-business-location">
+			<Navigation percentage={ navigationProgress } />
+			<div className="woocommerce-profiler-page__content woocommerce-profiler-business-location__content">
+				<Heading
+					title={ __(
+						'Where is your business located?',
+						'woocommerce'
+					) }
+					subTitle={ __(
+						"We'll use this information to help you set up payments, shipping, and taxes.",
+						'woocommerce'
+					) }
+				/>
+				<SelectControl
```

Comment:
The last thing! I found that there is a dropdown design. I think we should use the same style here though it's in "Tell us a bit about your store" section.

![Screenshot 2023-05-10 at 11 40 12](https://github.com/woocommerce/woocommerce/assets/4344253/a71df90b-017b-4ccd-820b-14f0200e71fb)

Y5pUYSJPsGEud1vknUZhi8-fi-739_60446

Reply:
Updated heading and dropdown style changes in https://github.com/woocommerce/woocommerce/pull/38019/commits/de8f406a693f2a36d91250ad3f3c24205c8b1767

Reply:
Thanks @moon0326, it seems that we need to make it look like the below when a country is selected. However, I'm not sure if it's easy to do that. If you want, we can do it in a follow-up PR.

![Screenshot 2023-05-12 at 11 22 15](https://github.com/woocommerce/woocommerce/assets/4344253/b620e4dd-36c2-4e58-a8cd-7c2b4f31625c)

Reply:
Yeah, let's do it in a follow up PR 👍